services/workflows-service/src/collection-flow/collection-flow.service.ts

```diff
@@ -324,4 +327,39 @@ export class CollectionFlowService {
       currentProjectId,
     );
   }
+
+  async uploadNewFile(projectId: string, workflowRuntimeDataId: string, file: Express.Multer.File) {
+    // upload file into a customer folder
+    const customer = await this.customerService.getByProjectId(projectId);
+
+    const runtimeDataId = await this.workflowService.getWorkflowRuntimeDataById(
+      workflowRuntimeDataId,
+      {},
+      [projectId],
+    );
+
+    const entityId = runtimeDataId.businessId || runtimeDataId.endUserId;
+
+    if (!entityId) {
+      throw new NotFoundException("Workflow does't exists");
+    }
+
+    // Remove file extension (get everything before the last dot)
+    const nameWithoutExtension = (file.originalname || generateRandomId(16)).replace(
+      /\.[^.]+$/,
+      '',
+    );
+    // Remove non characters
+    const alphabeticOnlyName = nameWithoutExtension.replace(/\W/g, '');
```

Comment:
same, this code is unnecessary

Reply:
let me explain what it does, i will keep the original file name concatenated with random string and also keeping the extension file name 

<img width="401" alt="image" src="https://github.com/ballerine-io/ballerine/assets/6435752/28f685ed-edb8-4ea6-93af-9b9d09915522">

Reply:
<img width="404" alt="image" src="https://github.com/ballerine-io/ballerine/assets/6435752/38c626d1-3729-4fdc-83e7-e2b91f625152">